starky/src/get_challenges.rs

```diff
@@ -79,27 +71,21 @@ where
 {
     // TODO: Should be used later in compression?
```

Comment:
Still relevant?

Reply:
Still unused for now (see `TODO: Deal with the compressed stuff.`)